Opserver/Views/Dashboard/Node.Services.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@using StackExchange.Opserver.Data.Dashboard
+@using StackExchange.Opserver.Models
+@model Node
+
+<h5 class="page-header">@Model.Services.Count.Pluralize("Service")</h5>
+<table class="table table-striped table-middle table-super-condensed table-responsive table-dashboard text-nowrap">
+    <thead>
+        <tr>
+            <th>Service</th>
+            <th>Description</th>
+            <th>State</th>
+            <th></th>
+            <th>Startup Type</th>
+            <th>Log On As</th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach (var i in Model.Services.OrderBy(i => i.DisplayName))
+        {
+            <tr class="@i.RowClass()">
+                <td title="@i.Caption
+                    last updated @(i.LastSync?.ToRelativeTime())" data-val="@(i.Id)">
```

Comment:
Titles in browsers are spaced like a `<pre>`, so I'd add title casing here and left-align for proper user-side UI, like this:
```html
                <td title="@i.Caption
Last Updated @(i.LastSync?.ToRelativeTime() ?? "Unknown")" data-val="@(i.Id)">
```